docs/getting_started/discover_ui.md

```diff
@@ -23,13 +42,13 @@ You can always return to the **My dashboard** screen by clicking the logo in the
 
 The left side menu allows you to move between important sections of the application.
 
-![Side menu](img/top_bar.png "Side menu")
+![Side menu](img/side_menu.png "Side menu")
 
 Depending on the product edition and your [permissions](../permission_management/permission_system.md), the top level sections on the leftmost pane may include, for example:
 
 - **Content**, which gives you access to the content Repository.
 It lets you navigate the Content Tree, create, edit, move, copy, delete content, etc.
-- [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]] **Site**, which enables you to create and edit block-based Pages and manage multiple websites.
+- [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]] **Site management**, which enables you to create and edit block-based Pages and manage multiple websites.
```

Comment:
What does it mean block-based pages?

Reply:
I have no idea, not my wording. I guess the author meant Pages with blocks